custom_components/senec/const.py

```diff
@@ -75,6 +79,21 @@ class ExtSensorEntityDescription(SensorEntityDescription):
 class ExtBinarySensorEntityDescription(BinarySensorEntityDescription):
     icon_off: str | None = None
 
+"""Supported number implementations"""
+NUMBER_SENSOR_TYPES = [
```

Comment:
can we call this WEB_NUMBER_SENYOR_TYPES (so we have always the info, what integration the constants are valid for)

Reply:
Changed to WEB_NUMBER_SENYOR_TYPES